src/modules/pos/validator/components/EditStake/index.js

```diff
@@ -97,6 +107,26 @@ const EditStake = ({ history, stakeEdited, network, staking }) => {
   const mode = validatorStake || staking[address] ? 'edit' : 'add';
   const titles = getTitles(t)[mode];
 
+  const isMonthly = selectedDuration === REWARD_DURATIONS.monthly;
+  const queryConfig = {
+    options: {
+      enabled: !!validator.address && !stakeAmount.error && Number(stakeAmount.value) > 0,
+    },
+    config: {
+      params: {
+        validatorAddress: validator.address,
+        stake: convertToBaseDenom(stakeAmount.value, token),
+        validatorReward: '0',
```

Comment:
should this be hardcoded to 0?

Reply:
this is a bug on service side.. we will remove this once service bug is fixed..